Dockerfile.alpine

```diff
@@ -17,12 +17,16 @@ RUN apk add --no-cache bzip2 ca-certificates coreutils libstdc++ tini w3m &&\
   apk del bzip2 w3m
 
 # create user, group, and set permissions
-RUN adduser -u 503 -g 503 -h ${TS_DIRECTORY} -D teamspeak &&\
-  mkdir /data &&\
-  chown -R teamspeak:teamspeak ${TS_DIRECTORY} /data
+RUN  echo "**** create teamspeak user and make our folders ****" && \
+  adduser -h ${TS_DIRECTORY} -s /bin/false -D teamspeak && \
+  PUID=${PUID:-911} && \
```

Comment:
I would still want the UID and GID to default to 503 otherwise it could break existing deployments.